core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -226,20 +243,21 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
                     return this.dotPageApiService
                         .save({
                             pageContainers,
-                            pageId: payload.pageId
+                            pageId: response?.payload.pageId,
+                            params: response?.params
```

Comment:
I didn't consider this further but I think we need to handle better this whole request, we shouldn't even check for duplicated content at this point of the process. 

But in this case there's a lack of error handling, because right now if there is no data the insert method will return `didInsert` as false and therefore landing in the logic to show an alert, that doesn't show the actual error, that will throw an alert that says to the user that the content is duplicated which is not true.

Reply:
After a quick discussion, we need to analyze this further and find another approach for this whole request.